sdk/appconfiguration/Azure.Data.AppConfiguration/samples/README.md

```diff
@@ -0,0 +1,73 @@
+# Azure App Configuration client SDK samples
+
+This directory contains samples for the Azure App Configuration client SDK.  Each sample illustrates one or more capabilities of the library.
+
+These include:
+ - [Hello World](#hello-world): Create and delete a configuration setting.
+ - [Hello World Extended](#hello-world-extended): Asynchronously create, update and delete configuration settings with labels.
+ - [Set Read Only](#set-read-only): Make a configuration setting read only, and then return it to a read write state.
+ - [Read Revision History](#read-revision-history): Read the revision history of a configuration setting that has been changed.
+ - [Get Setting If Changed](#get-setting-if-changed): Save bandwidth by using a conditional request to retrieve a setting only if it is different from your local copy.
+ - [Update Setting If Unchanged](#update-setting-if-unchanged): Prevent lost updates by using optimistic concurrency to update a setting only if your local updates were applied to the same version as the resource in the configuration store.
+ - [Mock Client](#mock-client): Mock a client for testing using the [Moq library][moq].
+
+ ## Prerequisites
+ To run these samples, you will need:
+ - An [Azure subscription][azure_sub].
+ - A [Configuration Store][configuration_store].
+ - To set the environment variable ``"APPCONFIGURATION_CONNECTION_STRING"`` to the connection string for your configuration store.
+
+ For more details regarding prerequisites, see the [README for this library][root_readme].
+
+## Hello World
+
+In this sample, we introduce ConfigurationClient and ConfigurationSetting, two core classes in this library.  ConfigurationClient is used to call the Azure App Configuration service -- each method call sends a request to the service's REST API.  ConfigurationSetting is the primary entity stored in a Configuration Store and represents a key-value pair you use to configure your application.  The sample walks through the basics of adding, retrieving, and deleting a configuration setting.
```

Comment:
I'd backtick all the type and method names, like `ConfigurationClient` and `ConfigurationSetting`.